Contents/mods/CommunityAPI/media/lua/client/ModSettingAPI/ModSandboxSetting.lua

```diff
@@ -0,0 +1,130 @@
+local jsonUtils = require("CommunityAPI/JsonUtils")
```

Comment:
Usage of require for CommunityAPI features like Utils should be like this
```lua
require("CommunityAPI")

local JsonAPI = CommunityAPI.Utils.Json

JsonAPI.Decode(...)
```

If the file path to that feature changed in the future direct require would need to be patched.